app/config/config.yml

```diff
@@ -6,7 +6,10 @@ services:
   multi_user.user_repository.collection:
     class: SumoCoders\FrameworkMultiUserBundle\User\UserRepositoryCollection
     arguments:
-      - ["@multi_user.user.repository"]
+      -
```

Comment:
This doesn't seem right

Reply:
Has been changed later (this commit was before the update of multiuserbundle)